playwright.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig, devices } from '@playwright/test';
+
+export default defineConfig({
+  testDir: './test/browser',
+  fullyParallel: true,
+
+  retries: 0,
+  reporter: 'list',
+  projects: [
+    {
+      name: 'chromium',
```

Comment:
here only chromium is tested, but the script in package.json downloads all engines. Should be consistent and don't download unused ones.